src/client/components/CompetitionCard.jsx

```diff
@@ -1,73 +1,80 @@
 import React, { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 
-import './styling/card.css'
+import './styling/card.css';
 
 import { CardTag } from './card/CardTag';
 import { PodiumParticipant } from './card/PodiumParticipant';
 
-export const CompetitionCard = ({ competitionId, user }) => {
-
-    console.log(competitionId)
-
-    //find competitors with the 3 highest scores
-    //(competitionId > many competitors > many participants > many placements > calculate and sum positionMapping)
-
-    let navigate = useNavigate();
-
-    const dummyData = [
-        {
-            name: "Steve",
-            nationality: "Great Britian",
-            score: 127
-        },
-        {
-            name: "Nathan",
-            nationality: "France",
-            score: 103
-        },
-        {
-            name: "Edward",
-            nationality: "Canada",
-            score: 94
+export const CompetitionCard = ({ competitionId, user, competition }) => {
+  const [seasonScores, setSeasonScores] = useState([]);
+  const [podium, setPodium] = useState([]);
+
+  console.log(competition);
```

Comment:
Make sure to remove this log before merging, you don't want these things in your deployed app